### PR TITLE
feat: [CDS-40851]:  added deletesourcebranch checkbox to merge pr step

### PR DESCRIPTION
**Pull request by sahithikolichala**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>
Screen Shot 2022-08-26 at 4 44 11 PM

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 1 Issues 
* Code issue 1 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10822)

**Comment by sahithikolichala** [comment]
retrigger titlecheck

File: src/modules/75-cd/components/PipelineSteps/CreatePrStep/CreatePrScript.tsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ export default function CreatePRScript(props: {
       </div>
       <div className={stepCss.divider} />
 
-      <Accordion className={cx(stepCss.accordion, css.overrideConfig)}>
+      <Accordion className={cx(stepCss.accordion)}>
```
Comment (mtran7) [inline src/modules/75-cd/components/PipelineSteps/CreatePrStep/CreatePrScript.tsx:97]: you could remove the cx if you're only referring to 1 className

**Review by mtran7 — DISMISSED** [review]

**Review by cjlee01 — DISMISSED** [review]

**Comment by mtran7** [comment]
Does the checkbox need to support runtime inputs/expressions? There is a component called FormMultiTypeCheckboxField if it does.

File: src/modules/75-cd/components/PipelineSteps/MergePrStep/MergePrStep.tsx [hunk 2]
```diff
@@ -142,6 +142,10 @@ function MergePRWidget(props: MergePrProps, formikRef: StepFormikFowardRef<StepE
                   />
                 )}
               </div>
+
+              <div className={cx(stepCss.formGroup, stepCss.sm)}>
+                <FormInput.CheckBox name="spec.deleteSourceBranch" label={'Delete Source Branch'} />
```
Comment (mtran7) [inline src/modules/75-cd/components/PipelineSteps/MergePrStep/MergePrStep.tsx:147]: put string into string.en.yaml

**Review by mtran7 — APPROVED** [review]

**Review by cjlee01 — APPROVED** [review]

**Review by tan-nhu — DISMISSED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by mtran7 — DISMISSED** [review]

**Review by vasuki-kalluri — APPROVED** [review]

**Review by vasuki-kalluri — APPROVED** [review]

**Review by mtran7 — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**